raistlin/tasks/views.py

```diff
@@ -136,3 +136,37 @@ def post(self, request):
                                 status=500)
         return JsonResponse({'status': 'ok',
                              'message': "Task removed correctly"}, safe=False)
+
+
+class RescheduleTask(View):
+    http_method_names = ['post']
+
+    @method_decorator(csrf_exempt)
+    def dispatch(self, *args, **kwargs):
+        return super(RescheduleTask, self).dispatch(*args, **kwargs)
+
+    def post(self, request):
+        task_data = json.loads(request.body)['taskData']
+        task_to_reschedule = {
+          'tasks': [{}]
+        }
+        task_to_reschedule['tasks'][0]['task_id'] = task_data['task_id']
+        task_to_reschedule['tasks'][0]['backend'] = task_data['backend']
+        task_to_reschedule['tasks'][0]['backend_args'] = \
+            task_data['backend_args']
+        task_to_reschedule['tasks'][0]['category'] = task_data['category']
+        task_to_reschedule['tasks'][0]['scheduler'] = \
+            task_data['scheduling_cfg']
+        if 'archive_cfg' in task_data:
```

Comment:
Maybe we didn't use the best name for this method in `arthur`. The thing is the post method doesn't need all this parameters. It only needs the id of the task.